util/init_context.go

```diff
@@ -17,6 +18,15 @@ func InitAmfContext(context *context.AMFContext) {
 	if configuration.AmfName != "" {
 		context.Name = configuration.AmfName
 	}
+	context.ServerIPv4 = os.Getenv(configuration.ServerIPv4)
+	if context.ServerIPv4 == "" {
+		logger.UtilLog.Warn("Problem parsing ServerIPv4 address from ENV Variable. Trying to parse it as string.")
```

Comment:
Should this be warning?
Because there should be lots of config setting using pure IP and walk through this warning.
Or maybe just a info for using IP or host env?

Reply:
In fact, I think Info would be enough.
However, I would keep the Warning in line 26 (which is reached if the field is completely empty/wrong in the config file).